enabler/src/components/payment-methods/card/utils.ts

```diff
@@ -52,11 +52,11 @@ const isFieldValid = (field: string) => {
   const input = getInput(field);
   switch (field) {
     case 'creditCardForm-cardNumber':
-      return input.value.replace(/\s/g, '').length === 16;
+      return input.value.replace(/\s/g, '').length >= 15;
```

Comment:
Suggest to check AMEX card pattern (when `getCardBand()`='amex', it should have 15-digit length)

Reply:
https://github.com/commercetools/connect-payment-integration-template/pull/162/commits/33e6d49dcdd75408884c38ec2cfc644eb9da13f3